docs/csharp/language-reference/operators/with-expression.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "with expression - C# reference"
+description: "Learn about a with expression that performs non-destructive mutation of C# records"
+ms.date: 11/10/2020
+f1_keywords:
+  - "with_CSharpKeyword"
```

Comment:
@Youssef1313 I've noticed you've worked recently in this area. Is that F1 keyword supported or going to be supported (so, I can leave it here)?

Reply:
I think it should work. But added a test for it in #49236 to confirm.

Reply:
@Youssef1313 thank you
dotnet/roslyn#49236